### PR TITLE
[xml] Update french.xml

### PR DESCRIPTION
**Pull request by Mateos81**
I see other translations have been marked accepted, I was waiting in case the C0/C1 PR was getting merged/inspired from, but let's open it anyway

### CODE REVIEW

**Review by donho — COMMENTED** [review]
1. `<TrayIcon>...</TrayIcon>` is missing.
2. `<Menu>...</Menu>` of `<Find title=...>` is missing.
3. Please consider the blow suggestions.

**Comment by donho** [comment]
@Mateos81 

C0/C1 PR won't be in this Release Candidate, but it will be in next release. That's why I start to merge the translation - when there will be no new translation entries.

File: PowerEditor/installer/nativeLang/french.xml [hunk 1]
```diff
@@ -1462,6 +1467,7 @@ Voulez-vous enregistrer vos modifications avant de changer de thème ?"/> <!-- H
             <ColumnVal name="Valeur"/>
             <ColumnHex name="Hexa"/>
             <ColumnChar name="Caractère"/>
+            <ColumnHtmlName name="Nom HTML"/>
             <ColumnHtmlNumber name="Numéro HTML"/>
             <ColumnHtmlName name="Code HTML"/>
```
[the remark this answers is not in the document]
Comment (Mateos81) [reply PowerEditor/installer/nativeLang/french.xml:1470]: Done!

> 1. `<TrayIcon>...</TrayIcon>` is missing.
> 
>     2. `<Menu>...</Menu>` of `<Find title=...>` is missing.

I'm not sure about these 2, as there is TrayIcon from line 425 to 432 in both files, and also Menu/Find title from line 436 to 483?

[the remark this answers is not in the document]
Comment (donho) [reply PowerEditor/installer/nativeLang/french.xml:1470]: Sorry my bad. Sorry about that!

**Pull request closed without merging**